File: program/convert-paise-to-rupees/ConvertPaiseToRupees.cs
using System;

class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter the paisa value : ");
    double paisa = Convert.ToDouble(Console.ReadLine());
    double rupees = paisa / 100 ;
    Console.WriteLine(paisa+" paisa is "+rupees+" rupees");
  }
}

File: program/convert-paise-to-rupees/convertPaiseToRupees.scala
import scala.io.StdIn._;

object paisaToRupees {
  def main(args:Array[String]):Unit={
    var paisa = readLine().toInt;
    println("Enter the paisa : "+paisa);
    var rupees = paisa / 100;
    println(paisa+" paisa is "+rupees+" rupees");
  }
}
